lib/testing/assert.fnl

```diff
@@ -18,6 +18,29 @@
                  (+ matches 1)
                  matches))) (.. message " instead got " (hs.inspect actual))))
 
+(fn all? [pred iter]
+  (var result true)
+  (each [k v (iter)]
+    (when (not (pred k v))
+      (set result false)))
+  result)
+
+(fn exports.table-eq? [first second message]
```

Comment:
Personally prefer args like expected and actual over first and second but not a big deal.

Reply:
You're right. I'll fix `not-eq?` to match